--- wmOptions.c~	Wed Sep 22 08:38:26 1999
+++ wmOptions.c	Tue Sep 28 11:17:20 1999
@@ -137,8 +137,8 @@
 	
 	if (stat("/usr/local/share/WindowMaker/",&pathStat)==0) {
 		strcpy(path,"/usr/local/share/WindowMaker/");
-	} else if (stat("/usr/share/WindowMaker/",&pathStat)==0) {
-		strcpy(path,"/usr/share/WindowMaker/");
+	} else if (stat("/usr/X11R6/share/WindowMaker/",&pathStat)==0) {
+		strcpy(path,"/usr/X11R6/share/WindowMaker/");
 	} else if (stat(path,&pathStat)!=0) path[0]=0;
 	if (path[0]==0) return;
 	strcpy(pathTemp,path);
